influxdb3_server/src/lib.rs

```diff
@@ -242,6 +246,56 @@ mod tests {
             "\n\nexpected:\n\n{:#?}\nactual:\n\n{:#?}\n\n",
             expected, actual
         );
+        // Test that we can query the output with a json output
+        let res = query(&server, "foo", "select * from cpu", "json", None).await;
+        let body = body::to_bytes(res.into_body()).await.unwrap();
+        let actual = std::str::from_utf8(body.as_bytes()).unwrap();
+        let expected = r#"[{"host":"a","time":"1970-01-01T00:00:00.000000123","val":1}]"#;
```

Comment:
The JSON output is actually supposed to be JSONL, which doesn't have the wrapping brackets `[]`. To really confirm that output, you'd ideal have at least two rows come back in the result so you can validate that it outputs one row per line.

Reply:
JSON Lines: https://jsonlines.org/

Reply:
We talked about this offline and decided to just open an issue for this -> https://github.com/influxdata/influxdb/issues/24654